config/conference.ts

```diff
@@ -8,26 +8,27 @@ import {
   ImportantDate,
 } from './types'
 import venue from './venue'
-import { zonedTimeToUtc } from 'date-fns-tz'
+import { zonedTimeToUtc, utcToZonedTime } from 'date-fns-tz'
 import { add, sub, set, toDate } from 'date-fns'
 
 const name = 'DDD Perth'
 const tagLine = `${name} is an inclusive non-profit conference for the Perth software community`
+const tz = '+08:00'
 
 const hideDate = false
 const ticketPurchasingOptions = TicketPurchasingOptions.OnSale
 const staticDate = '2022-09-10T08:00'
-const date = zonedTimeToUtc(staticDate, '+08:00')
+const date = utcToZonedTime(zonedTimeToUtc(staticDate, '+08:00'), tz)
```

Comment:
It still seems a bit strange that we take a static date in +8, then convert to UTC, then back to +8

I did consider that we could change where we format them rather than at the source, e.g. to `formatInConferenceTimeZone(date)`, but since what you've got seems to work, it might be overkill

Reply:
Would be interesting to see how it behaved around the boundaries of voting open, agenda published, etc, but that can be a bit annoying to test. Can use the `TestingControl` and add extra dates in there that are just before and just after the cut-offs, but I doubt we want that in there for the long term, so maybe something to test locally

Reply:
Yeah, I know. https://github.com/date-fns/date-fns/issues/489 has some explanation about ECMAScript Date() TZ issues.

Reply:
@bplowry Actually, you do raise a good point about boundary conditions there. I didn't check to see whether, for example, someone in a timezone ahead of Perth could now potentially see tickets/agenda details a few hours before people in Perth are able to. In theory, I think the constraints on the backend side would still prevent that, but it might produce a slightly broken frontend experience during the few hours-long time window. 🤔

Reply:
> Yeah, I know. [date-fns/date-fns#489](https://github.com/date-fns/date-fns/issues/489) has some explanation about ECMAScript Date() TZ issues.

We use `date-fns-tz` which has this nice function that seemed to work OK

https://github.com/marnusw/date-fns-tz#formatintimezone